components/rabbitmq/config/cloudify-rabbitmq.service

```diff
@@ -4,11 +4,16 @@ Wants=network-online.target
 After=network-online.target
 
 [Service]
-TimeoutStartSec=0
 Restart=on-failure
 EnvironmentFile=-/etc/sysconfig/cloudify-rabbitmq
+
+ExecStartPre=-/bin/sh -c "/sbin/rabbitmqctl status > /dev/null 2>&1"
```

Comment:
This isn't strictly required for this change, but fixes a bug described in https://bugzilla.redhat.com/show_bug.cgi?id=1059913